tests/rules/avoid-duplicate-actions-in-reducer.test.ts

```diff
@@ -1,176 +1,183 @@
+import type { InvalidTestCase } from '@typescript-eslint/experimental-utils/dist/ts-eslint'
 import { stripIndents } from 'common-tags'
 import path from 'path'
+import { test } from 'uvu'
+import type { MessageIds } from '../../src/rules/store/avoid-duplicate-actions-in-reducer'
 import rule, {
   avoidDuplicateActionsInReducer,
   avoidDuplicateActionsInReducerSuggest,
 } from '../../src/rules/store/avoid-duplicate-actions-in-reducer'
 import { ruleTester } from '../utils'
 
-ruleTester().run(path.parse(__filename).name, rule, {
-  valid: [
-    `
+const valid = [
+  `
+  export const reducer = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )`,
+  `
+  export const reducer = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )
+
+  export const reducerTwo = createReducer(
+    {},
+    on(abc, state => state),
+    on(def, state => state),
+    on(ghi, state => state),
+  )`,
+  // does not crash when no arguments present
+  `
+  export const reducer = createReducer(
+    {},
+    on(),
+  )`,
+]
+
+const invalid: InvalidTestCase<MessageIds, []>[] = [
```

Comment:
Interesting.. the `as const` should be sufficient here, however [`suggestions` doesn't accept a `readonly []`](https://github.com/typescript-eslint/typescript-eslint/blob/101abc72f0baced5cdf609d99418ad9f10830c73/packages/experimental-utils/src/ts-eslint/RuleTester.ts#L111) so we still can't use it right now.

For now, unfortunately, I think that's what we have as an option. Once my PR (https://github.com/typescript-eslint/typescript-eslint/pull/3844) has been reviewed/merged, we will be able to switch it to `as const` (this will only be possible in v2 though - when we update to `@typescript-eslint@v5`).

I'd just change it to `const invalid: Invalid TestCase <string, []>[] = [` because besides being a temporary solution, we don't really need to import `MessageIds` as it would give error in `ruleTester.run` if the type is different. Also, we're removing the `export` from `MessageIds` in #259 😄